tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
     nose
     pytest
     pytest-cov
-commands = py.test tests --cov={[base]name} --cov-fail-under=100
+commands = py.test tests --cov={envsitepackagesdir}/{[base]name} --cov-fail-under=100
```

Comment:
without `{envsitepackagesdir}`, it does not collect coverage info if there is no `__init__py` in `tests` (and that one came in conflict with something else I'm not eager to investigate, and was excessive anyway).